backend/Recodo.API/Startup.cs

```diff
@@ -52,6 +52,17 @@ public void Configure(IApplicationBuilder app, IWebHostEnvironment env)
             {
                 endpoints.MapControllers();
             });
+
+            InitializeDatabase(app);
+        }
+
+        private static void InitializeDatabase(IApplicationBuilder app)
+        {
+            using (var scope = app.ApplicationServices.GetService<IServiceScopeFactory>().CreateScope())
+            {
+                using var context = scope.ServiceProvider.GetRequiredService<RecodoDbContext>();
+                context.Database.Migrate();
+            };

```

Comment:
```suggestion
            using var scope = app.ApplicationServices.GetRequiredService<IServiceScopeFactory>().CreateScope();
            using var context = scope.ServiceProvider.GetRequiredService<RecodoDbContext>();
            context.Database.Migrate();
```
You can use 'using' declaration in both cases. The same for GetRequiredService